source/components/organisms/SummaryList/SummaryList.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useEffect } from "react";
 import PropTypes from "prop-types";
 import styled from "styled-components/native";
+import { findLastIndex } from "lodash";
```

Comment:
We do not have this dependency listed in our package.json, are you sure you aren't getting any warnings about that?

perhaps add
```js
lodash.findLastIndex 
```

with:
```js
yarn add lodash.findLastIndex
yarn add @types/lodash.findLastIndex
```

Reply:
As I don't want to add more ~~useless~~ dependencies, I've instead opted to write a small function for it instead.

Reply:
Even better!